styles/styles.css

```diff
@@ -0,0 +1,277 @@
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  border: 0;
+  padding: 0;
+  white-space: nowrap;
+  clip-path: inset(100%);
+  clip: rect(0 0 0 0);
+  overflow: hidden;
+}
+
+@font-face{
+font-family: "PT Sans";
+font-style: normal;
+font-weight: 400;
+scr: url("fonts/pt-sans-regular.woff2") format("woff2");
+}
+
+@font-face{
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 700;
+    scr: url("fonts/pt-sans-bold.woff2") format("woff2");
+    }
+
+ @font-face{
+    font-family: "PT Sans";
+    font-style: italic;
+    font-weight: 400;
+    scr: url("fonts/pt-sans-italic.woff2") format("woff2");
+    }
+
+@font-face{
+    font-family: "PT Sans";
+    font-style: italic;
+    font-weight: 700;
+    scr: url("fonts/pt-sans-italic.woff2") format("woff2");
+    }
+
+body{
+    font-family: "PT Sans",sans-serif;
+    font-size: 16px;
+    line-height: 24px;
+    color:#000000;
+    background-color:#ffffff;
+
+}
+
+.page-header{
+  color:#000000;
+  background-color:#ffffff;
+}
+
+
+.navigation-link{
+  font-family:"PT Sans",sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #000000;
+  background-color:#ffffff;
+  text-decoration: none;
+  text-transform: uppercase;
+
+}
+
+.navigation-link-button{
+  text-align: center;
+  color:#ffffff;
+  background-color:#756157;
+  Font-size: 16px;
+  Line-height: 20px;
+
+}
+
+
+.main-index{
+  color: #000000;
+  background-color: #ffffff;
+  background-image: url("../images/index-background.jpg");
+  background-size: 100% auto;
+  background-repeat: no-repeat;
+
+}
+
+.intro-description{
+  text-align: center;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 36px;
+  text-transform: uppercase;
+}
+
+.into-advantages{
+  text-align: center;
+  font-family: 'PT Sans';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 26px;
+}
+
+.advantages-list{
+  font-style: normal;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.advantages-item{
+  width: 400px;
+  height: 385px;
+  color: #FFFFFF;
+  background-color: #82B3D3;
+
+}
+
+.advantages-item-square{
+  color: #000000;
+  background-color: rgba(131, 179, 211, 0.12);
+}
+
+.advantages-item-tourists{
+  color: #000000;
+  background: rgba(131, 179, 211, 0.12);
+}
+
+.advantages-item-road{
+  color: #000000;
+  background-color: rgba(131, 179, 211, 0.2);
+}
+
+.advantages-slogan{
+  Font-size:30px;
+  Line-height: 36px;
+  text-align: center;
+}
+
+.advantages-slogan-description{
+  Font-size: 22px;
+  Line-height: 26px;
+  text-align: center;
+}
+
+.services-list{
+  font-style: normal;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.services-item{
+  width: 400px;
+  height: 385px;
```

Comment:
```suggestion
  min-height: 385px;
```
Иначе, если мы добавим побольше текста будет вот так:
![image](https://user-images.githubusercontent.com/17790771/221954185-09f10343-b10f-4ada-b49f-e11e05b87411.png)